docs/index.md

```diff
@@ -25,9 +25,8 @@ Wechaty.instance() // Singleton
 <p>Add/Del/Topic: <a href="https://github.com/wechaty/wechaty/issues/32">https://github.com/wechaty/wechaty/issues/32</a></p>
 </dd>
 <dt><a href="#Contact">Contact</a></dt>
-<dd><p>Class Contact
```

Comment:
Do not commit the new docs/index.md in this PR.

Because this is not necessary.

Please only commit the code file.